Source: debian patch
Upstream: No
Reason: set default terminal here is useless
--- lxpanel-0.5.1/data/default/panels/panel.in~	2009-08-03 15:21:45.752923980 +0800
+++ lxpanel-0.5.1/data/default/panels/panel.in	2009-08-03 21:17:09.508682430 +0800
@@ -53,9 +53,6 @@
             id=pcmanfm.desktop
         }
         Button {
-            id=gnome-terminal.desktop
-        }
-        Button {
             id=firefox.desktop
         }
     }
